Gemfile

```diff
@@ -33,8 +42,12 @@ group :development, :test do
   gem 'bixby', '2.0.0.pre.beta1' # bixby (i.e. rubocop presets)
```

Comment:
If you think it will be helpful, we can get a release of `rubocul` out so you can point at that instead. I was pointing at the github when I was started to use it, but it might be helpful to get a release out.

Reply:
Sure!  That sounds good to me.  Glad to switch as soon as we have the release out.